java/src/main/native/src/TableJni.cpp

```diff
@@ -2293,31 +2320,32 @@ JNIEXPORT jlongArray JNICALL Java_ai_rapids_cudf_Table_rollingWindowAggregate(
   CATCH_STD(env, NULL);
 }
 
-JNIEXPORT jlongArray JNICALL Java_ai_rapids_cudf_Table_timeRangeRollingWindowAggregate(
+JNIEXPORT jlongArray JNICALL Java_ai_rapids_cudf_Table_rangeRollingWindowAggregate(
     JNIEnv *env, jclass, jlong j_input_table, jintArray j_keys,
-    jintArray j_timestamp_column_indices, jbooleanArray j_is_timestamp_ascending,
+    jintArray j_orderby_column_indices, jbooleanArray j_is_orderby_ascending,
     jintArray j_aggregate_column_indices, jlongArray j_agg_instances, jintArray j_min_periods,
-    jintArray j_preceding, jintArray j_following, 
-    jbooleanArray j_unbounded_preceding, jbooleanArray j_unbounded_following, 
+    jintArray j_preceding, jintArray j_following,
```

Comment:
Hmm... I think we will need to change this part of the API. :/

Currently, `j_preceding` and `j_following` are arrays of integers. We'll need these to now be arrays of (pointers to) scalar objects.

Reply:
Yes, That's what Bobby Evans said in the previous comments, I am changing that. Thx mithun

Reply:
Done

Reply:
Ah, I see how you're doing it. I didn't realize that we could use `native_jpointerArray`. Thanks, I'll have to remember this.